packages/client/hmi-client/src/types/Types.ts

```diff
@@ -404,6 +404,7 @@ export interface Project extends TerariumAsset {
      */
     projectAssets: ProjectAsset[];
     codeAssets: Code[];
+    workflowAssets: Workflow[];
```

Comment:
FYI: This is going to cause a bit of confusion because the UI has a different typing for Workflow.


Side question: does this construct mean that whenever we load up a project we will automatically load its code, workflow, model... and what not? Is that going to be cost prohibitive? Or is this like a lazy load kind of thing where you only load stuff when you access them?

Reply:
On the first point, I'm wondering how we can align the two. I know that theres some weird typing things happening on the front end - i wonder if we can get away with just json blobs for the back end..

On the second point - I will investigate more, thats a good point.

Reply:
The problem is the lack of better dynamic typing support and inferencing in Java 😛 

So you can statically type out everything if we remove the idea of "Partials" from the UI-code, but that comes at a detriment to the front-end code ... unless, you want to move all the workflow related process logic down to the Java level.

Reply:
I'm going to raise the size of `Project` going to the front end at the next hmi-server call